include/sqlite_orm/sqlite_orm.h

```diff
@@ -1796,12 +1796,24 @@ namespace sqlite_orm {
             using columns_tuple = std::tuple<Args...>;
 
             columns_tuple columns;
-
+#ifndef SQLITE_ORM_AGGREGATE_BASES_SUPPORTED
             unique_t(columns_tuple columns_) : columns(std::move(columns_)) {}
+#endif
         };
 
         struct unindexed_t {};
 
+        template<class T>
+        struct prefix_t {
+            using value_type = T;
+
+            value_type value;
+
+#ifndef SQLITE_ORM_AGGREGATE_BASES_SUPPORTED
```

Comment:
This constructor isn't necessary in all cases.

Reply:
<img width="1070" alt="Снимок экрана 2024-05-06 в 11 07 51" src="https://github.com/fnc12/sqlite_orm/assets/6009640/4b0ec4b3-3617-4a0e-af97-b5bd2b4e1461">
are you sure?

Reply:
Yes, I am absolutely certain. There are no aggregate base classes involved. The error happens because you removed the `unique_t` constructor - the [branch build](https://ci.appveyor.com/project/fnc12/sqlite-orm/builds/49752631/job/w6wdg9utb2gf2e3h) of your previous commit has failed, too.

Reply:
oh now I see how it works. Thanks